tests/parser/test_call_graph_stability.py

```diff
@@ -0,0 +1,61 @@
+import random
+import string
+
+import pytest
+from hypothesis import given, settings
+from hypothesis.strategies import lists, text
+
+import vyper.ast as vy_ast
+from vyper.compiler.phases import CompilerData
+
+
+# random names for functions
+@settings(max_examples=20, deadline=1000)
+@given(
+    lists(text(alphabet=string.ascii_lowercase, min_size=1), unique=True, min_size=1, max_size=10)
+)
+@pytest.mark.fuzzing
+def test_call_graph_stability_fuzz(func_names):
+    def generate_func_def(func_name, i):
+        mutability = random.choice(["@pure", "@view", "@nonpayable", "@payable"])
```

Comment:
For the sake of documentation (you might add a code comment), depending on the outcome of the following issue and PR a `payable` decorator for an `internal` might throw:
- https://github.com/vyperlang/vyper/issues/3099
- https://github.com/vyperlang/vyper/pull/3123

Reply:
i mean i thought about this, but i don't think it's necessary -- first of all, we haven't decided on the outcome of the issue so it could be moot (a dead comment) and second, if we do ban payable on internal, there will be some tests to clean up anyways and this will just be one of them.

Reply:
Right, that argument makes sense. So let's keep it as-is.

Reply:
this is the right method to use from Hypothesis:
https://hypothesis.readthedocs.io/en/latest/data.html#hypothesis.strategies.sampled_from

Reply:
> this is the right method to use from Hypothesis: https://hypothesis.readthedocs.io/en/latest/data.html#hypothesis.strategies.sampled_from

done in fea5b2b